File: SEPTEMBER-2025/13-09-2025/p3541.py
class Solution:
    def maxFreqSum(self, s: str) -> int:
        count=defaultdict(int)
        vowels=["a","e","i","o","u"]
        maxv=0
        maxc=0
        for i in s:
            count[i]+=1
            if i in vowels:
                maxv=max(maxv,count[i])
            else:
                maxc=max(maxc,count[i])
        return maxv+maxc

        

File: SEPTEMBER-2025/13-09-2025/p3541.cpp
class Solution {
public:
    int maxFreqSum(string s) {
        int vowels = 0;
        int consonants = 0;
        int n = s.size();
        map<char, int> mpp;
        for (int i = 0; i < n; i++) {
            mpp[s[i]]++;
        }
        for (auto it : mpp) {
            char f = it.first;
            int s = it.second;
            if (f == 'a' || f == 'e' || f == 'i' || f == 'o' || f == 'u') {
                int count = s;
                vowels = max(count, vowels);
            } else {
                int count = s;
                consonants = max(count, consonants);
            }
        }
        return vowels + consonants;
    }
};